### PR TITLE
[Backport] AAP-19528 - Adding the subscription details to the access management guide (#1864)

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR backports the changes from #1864 to the 2.5 branch and includes the following:

* AAP-19528 - Adding the subscription details to the access management guide

* AAP-19528 updates for peer review changes

### CODE REVIEW

File: downstream/assemblies/platform/assembly-gateway-licensing.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-gateway-licensing"]
+= Managing {PlatformNameShort} licensing, updates and support
+
+:context: licensing-gw
+
+Ansible is an open source software project and is licensed under the GNU General Public License version 3, as described in the link:https://github.com/ansible/ansible/blob/devel/COPYING[Ansible Source Code]
```
Comment (sayjadha) [inline downstream/assemblies/platform/assembly-gateway-licensing.adoc:8]: Need a period at the end of the sentence.

File: downstream/modules/platform/proc-aap-activate-with-credentials.adoc [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 
 When {PlatformNameShort} launches for the first time, the {PlatformNameShort} Subscription screen automatically displays. You can use your Red Hat credentials to retrieve and import your subscription directly into {PlatformNameShort}.
 
-.Procedures
+[NOTE]
+====
+You are opted in for {Analytics} by default when you activate the platform on first time log in. This helps Red Hat improve the product by delivering you a much better user experience. You can opt out, after activating {PlatformNameShort}, by doing the following: 
+
+. From the navigation panel, select {MenuSetSystem}.
+. Uncheck the *Gather data for {Analytics}* option.
```
Comment (sayjadha) [inline downstream/modules/platform/proc-aap-activate-with-credentials.adoc:13]: Per IBM style guide -> Use "clear", not "uncheck" or "unselect", to refer to removing a mark from a check box.

File: downstream/modules/platform/proc-attaching-subscriptions.adoc [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+-----
+# subscription-manager list --consumed
+-----
+
+.Troubleshooting
+
+* If you are unable to locate certain packages that came bundled with the {PlatformNameShort} installer, or if you are seeing a `_Repositories disabled by configuration_` message, try enabling the repository by using the command:
++
+{PlatformName} {PlatformVers} for RHEL 8
```
Comment (sayjadha) [inline downstream/modules/platform/proc-attaching-subscriptions.adoc:69]: Is the text '{PlatformName} {PlatformVers} for RHEL 8' intentional? Seems a little out of place to me.

File: downstream/modules/platform/ref-controller-node-counting.adoc [hunk 4]
```diff
@@ -2,11 +2,11 @@
 
 = Node counting in licenses
 
-The {ControllerName} license defines the number of Managed Nodes that can be managed as part of a {PlatformName} subscription.
+The {PlatformNameShort} license defines the number of Managed Nodes that can be managed as part of your subscription.
 
 A typical license says "License Count: 500", which sets the maximum number of Managed Nodes at 500.
 
-For more information about managed node requirements for licensing, see https://access.redhat.com/articles/3331481.
+For more information about managed node requirements for licensing, see link:https://access.redhat.com/articles/3331481[How are "managed nodes" defined as part of the {PlatformName} offering]
```
Comment (sayjadha) [inline downstream/modules/platform/ref-controller-node-counting.adoc:9]: Period at the end of the sentence.

**Review by sayjadha — COMMENTED** [review]
@dcdacosta : Some minor comments for your consideration, rest LGTM. Now, since this is a backport, and I don't want to hold you up, I can merge this PR and you can incorporate the edits in a new PR if that works better. Let me know.

**Review by sayjadha — APPROVED** [review]
Merging this PR, @Donna will incorporate the edits in an in-progress PR.

File: downstream/modules/platform/proc-attaching-subscriptions.adoc (hunk 3 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-attaching-subscriptions.adoc:69]: Yes, this is intentional. :D

File: downstream/assemblies/platform/assembly-gateway-licensing.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/assemblies/platform/assembly-gateway-licensing.adoc:8]: Fixed in https://github.com/ansible/aap-docs/pull/1883

File: downstream/modules/platform/ref-controller-node-counting.adoc (hunk 4 above)
Comment (dcdacosta) [reply downstream/modules/platform/ref-controller-node-counting.adoc:9]: Fixed in https://github.com/ansible/aap-docs/pull/1883

File: downstream/modules/platform/proc-aap-activate-with-credentials.adoc (hunk 2 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-aap-activate-with-credentials.adoc:13]: Fixed in https://github.com/ansible/aap-docs/pull/1883

**Pull request merged**